### PR TITLE
[ESSNTL-5265] Store groups information when mapped system is added or updated

### PR DESCRIPTION
**Pull request by petrblaho**
## Secure Coding Practices Checklist Link
- https://github.com/RedHatInsights/secure-coding-checklist

## Secure Coding Checklist
- [x] Input Validation
- [ ] Output Encoding
- [ ] Authentication and Password Management
- [ ] Session Management
- [ ] Access Control
- [ ] Cryptographic Practices
- [ ] Error Handling and Logging
- [ ] Data Protection
- [ ] Communication Security
- [ ] System Configuration
- [ ] Database Security
- [ ] File Management
- [ ] Memory Management
- [x] General Coding Practices

### CODE REVIEW

**Comment by petrblaho** [comment]
[ESSNTL-5265](https://issues.redhat.com/browse/ESSNTL-5265)

**Comment by petrblaho** [comment]
Some tech debt will be solved in https://issues.redhat.com/browse/RHINENG-1701

File: system_baseline/models.py [hunk 1]
```diff
@@ -78,10 +78,10 @@ def validate_existing_system(self, system_id):
                     "System {} already associated with this baseline".format(system_id)
                 )
 
-    def add_mapped_system(self, system_id):
+    def add_mapped_system(self, system_id, groups=None):
         self.validate_existing_system(system_id)
         new_mapped_system = SystemBaselineMappedSystem(
-            system_id=system_id, account=self.account, org_id=self.org_id
+            system_id=system_id, account=self.account, org_id=self.org_id, groups=groups
```
Comment (vkrizan) [inline system_baseline/models.py:84]: How do we update groups on a SystemBaselineMappedSystem table?

Comment (vkrizan) [reply system_baseline/models.py:84]: Never mind. Found it, ... updating within HSP by calling baselines internally https://issues.redhat.com/browse/ESSNTL-5248

**Review by vkrizan — APPROVED** [review]

**Pull request merged**